scheduler/scheduler_dfs.go

```diff
@@ -193,14 +194,30 @@ func (s *syncClient) resolveResourcesDfs(ctx context.Context, table *schema.Tabl
 		resolvedResources <- resource
 		for _, relation := range resource.Table.Relations {
 			relation := relation
+			// Depending on the source, rate limiting might be done on the table basis (GCP) or client + table (AWS)
```

Comment:
Can you explain the reason to chose between the options? Why GCP is table only and AWS both table + client?

Reply:
GCP applies the rate limit to the Project where the callers identity resides. So if a single caller is calling the same API across 50 projects it all counts towards a single rate limit.

In AWS the rate limit applies to the account+region where the resource resides and doesn't matter about the callers identity

Reply:
Maybe the API should be pass a function that generates the key with the default to use table name? I'm wondering what would be a good default here for other plugins